projects/1.20-fabric/assets/chatpatches/chatpatches/lang/zh_cn.json

```
@@ -8,63 +8,70 @@
 	"text.chatpatches.help.faq": "常见问题解答 (FAQ)，点击这里！",
```

```suggestion
	"text.chatpatches.help.faq": "常见问题解答(FAQ)，点击这里！",
```
看具体显示效果了，如果没问题就不用改成汉语标点

---

好的